src/Service/Resolvers/CosmosClientProvider.cs

```diff
@@ -117,5 +127,11 @@ private void ParseCosmosConnectionString()
             _accountEndpoint = dbConnectionStringBuilder.ContainsKey("AccountEndpoint") ? (string)dbConnectionStringBuilder["AccountEndpoint"] : null;
             _accountKey = dbConnectionStringBuilder.ContainsKey("AccountKey") ? (string)dbConnectionStringBuilder["AccountKey"] : null;
         }
+
+        private static string GetCosmosUserAgent()
+        {
+            return Environment.GetEnvironmentVariable(DAB_APP_NAME_ENV) ?? DEFAULT_APP_NAME;
```

Comment:
Can we add a second test case that covers the using the environment variable?

Reply:
Yes this uses env var already right? If env is not set, then it picks up the default name

Reply:
Ah sorry misread. Yeah will try to come up with one

Reply:
Done